src/enums.py

```diff
@@ -0,0 +1,18 @@
+# List of URLs for news sites with their respective 'main body of text' HTML tags
```

Comment:
If your find_article_parent_class function works in the utils file, what is this ACCEPTED_NEW_SITES dictionary for? Could it be deleted, going forward? (I appreciate that we wouldn't want to lose all the hard coded stuff, given the time spent to find those HTML tags! But theoretically speaking, will it be needed?)

Reply:
I removed this code from the utils file, utils now imports it from this enum. This is a better way of doing it since this is more configuration data than code, so you want those things to go into different files.